tests/packages/Microsoft.Bot.Components.Telephony.Tests/Integration/RecordingTests/StartRecording_HappyPath.test.dialog

```diff
@@ -7,22 +7,71 @@
             "$kind": "Microsoft.Test.UserSays",
             "text": "Hello I'm Calculon"
         },
+        {
+            "$kind": "Microsoft.Test.AssertReplyActivity",
```

Comment:
I have left the file names unchanged to allow for easy diff. Will rename to Recording_HappyPath.. in a next iteration.